tasks_downstream.py

```diff
@@ -46,8 +47,15 @@
         "build"
     ]["args"]["ODOO_VERSION"]
 )
-DOCKER_COMPOSE_CMD = f"{shutil.which('docker')} compose" or shutil.which(
-    "docker-compose"
+
+docker_compose_v2 = (
+    subprocess.run([which("docker"), "compose"], capture_output=True).returncode == 0
```

Comment:
Don't take my word for granted :sweat_smile: 
```suggestion
    subprocess.run([shutil.which("docker"), "compose"], capture_output=True).returncode == 0
```